html/includes/functions.inc.php

```diff
@@ -704,7 +704,62 @@ function generate_port_link($port, $text = null, $type = null, $overlib = 1, $si
     }
 }//end generate_port_link()
 
+function generate_sensor_link($args, $text = null, $type = null)
```

Comment:
usually on these functions the array argument is passed directly to the graph link function.  Also, you should probably pass the sensor as a separate parameter.

Reply:
I used the same structure of code as ports, accesspoints... Except that the var is named $args instead of $sensor (the content is anyway $tmp_alerts). 
I would prefer to avoid touching all of that code. I think that if we refactor all, the best would be to use the Alert class and have the templating system do that job completely. 
So what change would you exactly expect at this step ? only renaming $args to $sensor ?